### ISSUE TITLE
issue with cmdLineParse,main

### ISSUE THREAD

**Issue by totaibi** [aria-tools/ARIA-tools#328]
Dear ARIA team,

I installed the last version of ARIA and got the following error:

```
Traceback (most recent call last):
  File "/Users/Thamer/ARIA-tools/tools/bin/ariaTSsetup.py", line 11, in <module>
    from ARIAtools.tsSetup import cmdLineParse,main
ImportError: cannot import name 'cmdLineParse' from 'ARIAtools.tsSetup' (/Users/Thamer/development/ARIA-tools/tools/ARIAtools/tsSetup.py)
```

Your support is appreciated

**Comment by totaibi** [comment]
Problem resolved by updating the requirements 

Thanks
